src/components/AddToCart/AddToCart.stories.js

```diff
@@ -34,13 +35,15 @@ const currentMenu = {
 };
 
 export default {
-  title: 'Components/AddToCart',
+  title: 'Component/AddToCart',
   component: AddToCart,
+  decorators: [
+    AddToCart => (
+      <MemoryRouter>
+        <AddToCart />
+      </MemoryRouter>
+    ),
```

Comment:
I think this needs to be named `Story`:
```suggestion
   (Story) => (
      <MemoryRouter>
        <Story />
      </MemoryRouter>
    ),
```